src/main/resources/META-INF/mule-artifact/mule-artifact.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
My first attempt was to generate this file by adding parameters into the service pom. I had that working form the plugin side, but then I found issues when I had to add support for the new service into the test runner.
I'm not against generating this files, but note that they were not generated before with the old model. I can create a new issue to do it in a separate task